lib/Mailer/NotificationMailerAdapter.php

```diff
@@ -25,6 +25,7 @@
 use OCP\IUserManager;
 use OCP\ILogger;
 use OCP\IURLGenerator;
+use OCA\Notifications\Mailer\NotificationMailer;
```

Comment:
This is an unused "use" - I will backport the recent commit from master...

Reply:
More that unused, unneeded. I'd prefer to keep things as explicit as possible, but I'm fine removing this to reduce the amount of changes.